### PR TITLE
Upgrade py, beautifulsoup4 and pytest

### PR DESCRIPTION
**Pull request by pallix**
Upgraded versions are python3.5 compatible.

### CODE REVIEW

**Comment by nidico** [comment]
Still works on python 3.4, merging.

File: src/adhocracy_core/versions.cfg [hunk 1]
```diff
@@ -181,7 +181,7 @@ pyramid-tm = 0.11
 # Required by:
 # pytest-pyramid==0.1.1
 # pytest-splinter==1.2.9
-pytest = 2.6.4
+pytest = 2.7.3
```
Comment (joka) [inline src/adhocracy_core/versions.cfg]: why don't you update to pytest 2.8?

Comment (pallix) [inline src/adhocracy_core/versions.cfg]: true, it should be 2.8

Comment (nidico) [inline src/adhocracy_core/versions.cfg]: There's 2.8.2 now.

**Pull request merged**